source/android/adaptivecards/src/main/res/drawable/adaptive_card_ic_baseline_search_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
```

Comment:
in general, when adding visual elements, please share a screenshot of how the rendered visual looks like